clock.go

```diff
@@ -0,0 +1,11 @@
+package main
```

Comment:
what does this file do? 

Might not hurt to add a comment to the top of it explaining it's purpose ?